packages/app/src/__deprecated__/renderer/locales/default/en-US.json

```diff
@@ -266,10 +265,10 @@
   "module.connecting.errorSyncModalDescription": "Please try to reconnect your device",
   "module.connecting.errorSyncModalHeaderTitle": "Error",
   "module.connecting.errorSyncModalTitle": "Unfortunately, synchronization failed",
-  "module.connecting.criticalBatteryLevelModalHeaderTitle": "MuditaOS",
```

Comment:
Are you sure that the removed translations are not in use anymore?

Reply:
it's a good question. Why it's removed? 🤔